plaso/parsers/locate.py

```diff
@@ -18,7 +18,8 @@ class LocateDatabaseEvent(events.EventData):
   """Linux locate database (updatedb) event data.
 
   Attributes:
-    paths (list[str]): paths of the locate database (updatedb) entry.
+    contents (list[str]): contents of the locate database (updatedb) entry.
```

Comment:
Rename contents to entries for consistency with other event data definitions.

Reply:
Done